style.css

```diff
@@ -0,0 +1,824 @@
+
+  .header__inner {
+    width: 100%;
+    height: 89px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+    z-index: 3;
+    height: 89px;
+    width: 89px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 36.7px;
+    margin-bottom: 7px;
+  }
+
+  .header__logo p {
+    margin: 15px 4px;
+    color: black;
+    font-size: 38px;
+    letter-spacing: 4px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    max-width: 750px;
+    height: 569px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 63px;
+    line-height: 67px;
+    padding-left: 36px;
+    letter-spacing: 3px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 51px;
+    height: 25px;
+    left: 45%;
+    top: 624px;
+    display: block;
+  }
+}
+
+@media (max-width: 422px) {
+  .first-container {
+    padding: 71px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 422px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 80px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+    z-index: 3;
+    height: 80px;
+    width: 80px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 32.7px;
+    margin-bottom: 7px;
+  }
+
+  .header__logo p {
+    margin: 11px 4px;
+    color: black;
+    font-size: 33px;
+    letter-spacing: 2px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    height: 462px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 55px;
+    line-height: 57px;
+    padding-left: 36px;
+    letter-spacing: 1px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 43px;
+    height: 23px;
+    left: 45.5%;
+    top: 511px;
+    display: block;
+  }
+}
+
+@media (max-width: 360px) {
+  .first-container {
+    padding: 22px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 360px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 65px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
+    z-index: 3;
+    height: 65px;
+    width: 65px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 27.7px;
+    margin-bottom: 7px;
+  }
+
+  .header__logo {
+    margin: 4px 28px;
+  }
+
+  .header__logo p {
+    margin: 10px 0;
+    color: black;
+    font-size: 28px;
+    letter-spacing: 2px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    height: 381px;
+    letter-spacing: 5px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 46px;
+    line-height: 49px;
+    padding-left: 28px;
+    letter-spacing: 2px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)